crypto/ec/ec_mult.c

```diff
@@ -206,8 +206,8 @@ int ec_scalar_mul_ladder(const EC_GROUP *group, EC_POINT *r,
      */
     cardinality_bits = BN_num_bits(cardinality);
     group_top = bn_get_top(cardinality);
-    if ((bn_wexpand(k, group_top + 1) == NULL)
-        || (bn_wexpand(lambda, group_top + 1) == NULL)) {
+    if ((bn_wexpand(k, group_top + 2) == NULL)
+        || (bn_wexpand(lambda, group_top + 2) == NULL)) {
```

Comment:
Is this an unrelated problem?

Reply:
Actually I don't understand how k + 2*cardinality could need more than group_top + 1
BN_words, assuming that each BN_word has at least 32 bits or so.

Reply:
Ah as matt explained, this makes sense, consider this hunk approved.